### PR TITLE
Update GettingStarted.md

### PR DESCRIPTION
**Pull request by Gorild**
## 📥 Proposed changes

Minor edits and language changes

## ☑️ Submission checklist

-   [x] I have read the [CONTRIBUTING](https://github.com/fremtind/jokul/blob/master/CONTRIBUTING.md) doc
-   [x] `yarn build` works locally with my changes
-   [x] I have added tests that prove my fix is effective or that my feature works
-   [x] I have added necessary documentation (if appropriate)

## 💬 Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did, what alternatives you considered, etc...

### CODE REVIEW

File: guides/GettingStarted.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
+## Komme i gang
 
-Jøkul kan brukes på to måter, rene stilark eller React komponenter. Alle css-stilark kommer både vanlig og minifisert. Vi anbefaler å bruke den minifiserte.
+Du kan bruke Jøkul på to måter, med rene stilark eller med React-komponenter. Alle CSS-stilark finnes i både vanlig og minifisert utgave. Vi anbefaler å bruke den minifiserte.
 
-### Quick start
+### Komme i gang med React-komponenter
 
-For å ta ibruk en Jøkul React komponent så trenger man å laste inn både stil og komponent. Stilen kan lastes direkte i React om man har satt opp en css loader i pakke systemet, eller det kan legges inn i en av dine `.less` eller `.scss` filer.
+Hvis du vil ta i bruk en Jøkul React-komponent, trenger du å laste inn både stilen og komponenten. Hvis du har satt opp en CSS loader i pakkesystemet, kan du kan laste ned stilen direkte i React. Du kan også velge å legge den inn i en av  dine `.less`- eller `.scss`-filer.
```
Comment (piofinn) [inline guides/GettingStarted.md]: skal det ikke være bindestrek etter forkortelser? (CSS-loader)

File: guides/GettingStarted.md [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
 
-> Du bør `core.min.css` fra `@fremtind/jkl-core` for å få normalisert stilen og få tilgang på rot-stil. Trenger bare gjøres en gang.
+> For å normalisere stilen og få tilgang til rotstilen, kjører du `core.min.css` fra `@fremtind/jkl-core`. Det trenger du bar å gjøre en gang.
```
Comment (piofinn) [inline guides/GettingStarted.md]: ~~bar~~ --> **bare**

Comment (piofinn) [reply guides/GettingStarted.md]: Også: ~~kjører du~~ --> **laster du inn**

File: guides/GettingStarted.md (hunk 1 above)
Comment (Gorild) [reply guides/GettingStarted.md]: If both the acronym and the following word is in English, you do not have to add a hyphen. If the acronym is supposed to go with a Norwegian word, you need the hyphen.

**Pull request merged**